src/syncgitlab2msproject/sync.py

```diff
@@ -129,7 +129,7 @@ def update_task_with_issue_data(
             type_setter = task_type_setter(issue)
             type_setter.set_task_type_before_sync(task, is_add)
             task.name = issue.title
-            task.notes = issue.description
+            #task.notes = issue.description
```

Comment:
Why did you remove this?

Make this an option instead. So users can disable the functionality.